.gitattributes

```diff
@@ -0,0 +1,7 @@
+/tests export-ignore
```

Comment:
Wait, this is not really correct... It is `test` instead (unless you would like to a commit to move things around)

Reply:
ouch, sorry :sweat_smile: